figma/mod.ts

```diff
@@ -20,6 +20,7 @@ export type AppContext = FnContext<State, Manifest>;
  * @name FIGMA
  * @title Figma
  * @description A Deco app to interact with the Figma APIs with strongly typed responses
+ * @logo https://ih1.redbubble.net/image.4053581522.4386/raf,360x360,075,t,fafafa:ca443f4786.jpg
```

Comment:
I would avoid such non figma related company

> Redbubble is a global online marketplace for print-on-demand products based on user-submitted artwork. The company was founded in 2006 in Melbourne, Australia, and also maintains offices in San Francisco and Berlin.

Wikimedia allow hotlink, but they can be vandalized:

- https://upload.wikimedia.org/wikipedia/commons/a/ad/Figma-1-logo.png

Ideally, I would try a link from figma.